modules/build/src/test/scala/scala/build/tests/BuildOptionsTests.scala

```diff
@@ -124,6 +124,22 @@ class BuildOptionsTests extends munit.FunSuite {
     )
   }
 
+  test("Scala 3.1.2-RC1 works") {
+
+    val options = BuildOptions(
+      scalaOptions = ScalaOptions(
+        scalaVersion = Some("3.1.2-RC1"),
+        scalaBinaryVersion = None,
+        supportedScalaVersionsUrl = None
```

Comment:
No need of the explicit `None`s
```suggestion
        scalaVersion = Some("3.1.2-RC1")
```

Reply:
Good idea!